test/e2e/scenario/recorder/recorder.scenario.ts

```diff
@@ -658,6 +661,65 @@ describe('recorder', () => {
         expect(segment.records.slice(3).every((record) => record.type !== RecordType.FullSnapshot)).toBe(true)
       })
   })
+
+  describe('frustration records', () => {
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review (if it exists :^)  -->
**👏 praise:** Nice to see E2E tests!